README.md

```diff
@@ -1743,7 +1743,6 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 - [Lowdefy](https://www.lowdefy.com/) - Build internal tools, BI dashboards, admin panels, CRUD apps and workflows in minutes using YAML / JSON on an self-hosted, open-source platform. Connect to your data sources, host via Serverless, Netlify or Docker. ([Source Code](https://github.com/lowdefy/lowdefy)) `Apache-2.0` `Nodejs`
 - [ML Workspace](https://github.com/ml-tooling/ml-workspace) - All-in-one web-based IDE for machine learning and data science. `Apache-2.0` `Docker`
 - [Motor Admin](https://www.getmotoradmin.com/) - No-code admin panel and business intelligence software - search, create, update, and delete data entries, create custom actions, and build reports. ([Demo](https://motor-admin.herokuapp.com/demo/), [Source Code](https://github.com/motor-admin/motor-admin)) `AGPL-3.0` `Ruby`
-- [Regexr](https://regexr.com/) - RegExr is a HTML/JS based tool for creating, testing, and learning about Regular Expressions. ([Source Code](https://github.com/gskinner/regexr)) `MIT` `Nodejs`
```

Comment:
This was the last software in it's own "category". I could not find any FOSS/self-hosted alternative. I still use https://regexr.com/ occasionally and it works. But considering the rather large codebase and absence of maintenance, OK to remove it.

```
$ cloc regexr/
     156 text files.
     156 unique files.                                          
       9 files ignored.

github.com/AlDanial/cloc v 1.86  T=0.12 s (1229.8 files/s, 294675.2 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
JavaScript                      49           2182           2597          14158
JSON                             4              0              0           9225
PHP                             32            600            600           2141
Sass                            22            446             43           1905
SVG                             33              0             13            541
HTML                             2             36              8            460
CSS                              3             43             23            276
SQL                              1             21             31             74
Markdown                         1              9              0             22
XML                              1              0              0              9
-------------------------------------------------------------------------------
SUM:                           148           3337           3315          28811
-------------------------------------------------------------------------------
```